client/Pages/EventTypeCreate/Models.elm

```diff
@@ -84,9 +89,12 @@ defaultValues =
     , ( FieldPartitionStrategy, partitionStrategies.random )
     , ( FieldPartitionsNumber, "1" )
     , ( FieldPartitionKeyFields, emptyString )
+    , ( FieldOrderingKeyFields, emptyString )
     , ( FieldRetentionTime, toString defaultRetentionDays )
     , ( FieldSchema, defaultSchema )
     , ( FieldCompatibilityMode, compatibilityModes.forward )
+    , ( FieldAudience, audiences.component_internal )
```

Comment:
This field has no default. Does this mean it will be `component_internal` if the user don't change it?

Reply:
Yes, exactly, and I don't see any problem with it. This kind of decision should not stop user fro what he wants to do right now "I want to create ET in two clicks". This field can be changed any time later. So I choose the safest option "internal" only, so later you can expose it gradually to a broader and broader audience.

Reply:
👍